powerplantmatching/package_data/config.yaml

```diff
@@ -16,31 +16,34 @@ main_query: "Name != '' and (lat >= 30 or lat != lat)"
   # Queries are combined with `main_query` with an `and` operator
-  - ENTSOE: Fueltype != 'Solar'
-  - GEO: Fueltype != 'Solar'
-  - GPD: Fueltype != 'Solar'
-  - JRC: Fueltype != 'Solar'
-  - OPSD: Country != "Spain" and Fueltype != 'Hard Coal' and Fueltype != 'Solar'
-  - BEYONDCOAL: Fueltype != 'Solar'
-  - WIKIPEDIA: Fueltype != 'Solar'
-  - GEM
-  - MASTR
+  - ENTSOE: not (Country == 'Germany' and Fueltype  == 'Wind') # wind is per turbine rather than park in MASTR and unsuitable for matching
+  - GEO: Capacity >= 1 and not (Country == 'Germany' and Fueltype  == 'Wind') and Fueltype not in ['Oil', 'Nuclear'] and not (Country in ['Bulgaria', 'Slovakia'] and Fueltype == 'Hard Coal')
+  - GPD: Capacity >= 1 and not (Country == 'Germany' and Fueltype  == 'Wind') and not (Country in ['Czechia', 'Bulgaria', 'Romania'] and Fueltype == 'Hard Coal') and Fueltype != 'Nuclear'
+  - JRC: Capacity >= 1 and not (Country == 'Germany' and Fueltype  == 'Wind') # do not match small hydro
+  - OPSD: not (Country == 'Germany' and Fueltype  == 'Wind') and ((Capacity >= 1 and Fueltype != 'Solar') or Capacity >= 3) and not (Country == 'Spain' and Fueltype == 'Hard Coal') and not (Country == 'Italy' and Fueltype == 'Natural Gas')
+  - BEYONDCOAL
+  - GEM: Capacity >= 3 and not (Country == 'Germany' and Fueltype  == 'Wind')
+  # do not match units below 1 MW (2 MW for biogas, 3 MW for solar), exclude wind in Germany from any matching
+  - MASTR: (Fueltype != 'Wind') and ((Fueltype == 'Solar' and Capacity >= 3) or (Fueltype == 'Biogas' and Capacity >= 2) or (Fueltype not in ['Solar', 'Biogas'] and Capacity >= 1))
+  - EESI
+  - GHR
 
-# fully_included_sources, these sources are included even without match to the final dataset
+# # fully_included_sources, these sources are included even without match to the final dataset
 fully_included_sources:
   # Make individual queries for the datasets
-  - ENTSOE: (Country not in ['Switzerland', 'Ireland', 'Albania', 'Greece', 'Czech Republic', 'Bulgaria', 'United Kingdom', 'Italy', 'Serbia'] and not (Country == 'Spain' and Fueltype == 'Hydro')) or (Fueltype == 'Geothermal')
-  - GEO: (Country == 'Spain' and Fueltype == 'Natural Gas')
-  - GPD: Country in ['Finland', 'Spain']
-  - JRC: Country not in ['Switzerland', 'Albania', 'United Kingdom', 'Norway']
-  - OPSD: Country not in ['Switzerland', 'Italy', 'Spain', 'Norway', 'Austria']
+  - GEM: not (Country == 'Germany' and Fueltype in ['Solar', 'Wind']) # wind and solar in Germany is covered by MASTR
+  - EESI: Fueltype != 'Hydro' and not (Country == 'Germany' and Fueltype == 'Battery') # battery in Germany is covered by MASTR
+  - MASTR: Capacity >= 0.1 and Fueltype != 'Nuclear'
+  - OPSD: Country != 'Germany' and Capacity < 1 and Capacity >= 0.1 and Fueltype == 'Hydro' # take small hydro outside Germany from OPSD (highest coverage)
   - BEYONDCOAL
-  - GEM: Country != 'Germany' or Fueltype == 'Solar'
-  - MASTR
+  - JRC: Country in ['Italy', 'Croatia', 'Serbia', 'Slovakia']
```

Comment:
We should document the choices and learning that lead to the values here somehow, else they will seem arbitrary in the future.